### DISCUSSION TITLE
how to install older version of lief for newer version of python

### DISCUSSION THREAD

**Discussion by PiRK** [lief-project/LIEF#1129] [General]
I'm working with a script that requires lief==0.13.2. Since Ubuntu 24.04 has python 3.12 and there are no installable packages for this python version pre 0.14 on PyPi, I'm not sure how to install it.

Is there a simple way for me to build a cp312 wheel (.whl) file myself? I'm willing to clone the repo, checkout the correct commit, I'm just unsure about what the build command is.

**Comment by romainthomas** [comment]
Yes you will need to compile yourself unless there is a strong need (DM me in this case)
